src/main/java/org/example/Main.java

```diff
@@ -3,5 +3,6 @@
 public class Main {
     public static void main(String[] args) {
         System.out.println("Hello world!");
```

Comment:
Capitalize "World"

Reply:
That was IntelliJ that did it that way lol